tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.AppSec.cs

```diff
@@ -39,13 +39,6 @@ internal class AppSec
             /// </summary>
             public const string ExtraHeaders = "DD_APPSEC_EXTRA_HEADERS";
 
-            /// <summary>
-            /// Specifies if the AppSec traces should be explicitly kept or dropped.
-            /// Default is true, to keep all traces, false means drop all traces (by setting AutoReject as sampling priority).
-            /// For internal testing only.
-            /// </summary>
-            internal const string KeepTraces = "DD_APPSEC_KEEP_TRACES";
```

Comment:
I know it wasn't  not documented but beware, now that you are GA, I know some people that would say it's a breaking change 🤐  😉